_data/navigation.yml

```diff
@@ -194,6 +196,8 @@ t:
             url: /cn/docs/quick-start-advance/distributed-tracing/
       - title: "数据一致性解决方案"
         url: /cn/docs/quick-start-dataconsistency/
+      - title: "微服务 Meetup：Apache ServiceComb (incubating) Day"
```

Comment:
The docs link is for the quick start, we can add the link to the news section on the home page. 
Please check out  https://github.com/apache/incubator-servicecomb-website/blob/master/_pages/home.md  for more information.

Reply:
Don't worry, I can fix it after merged the PR. 
Just pay attention next time.